test/helpers/getVerificationId.js

```diff
@@ -0,0 +1,14 @@
+import { User } from '../../models';
```

Comment:
@leksyib let's form the habit of creating bug fixes for previous mistakes. Working on different scopes in a story is not ideal.
Please pass this to the team.